lisa/tools/fio.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from decimal import Decimal
+from enum import Enum
+from typing import Any, Dict, List, cast
+
+from lisa.executable import Tool
+from lisa.notifier import DiskPerformanceMessage
+from lisa.operating_system import Debian, Posix, Redhat, Suse
+from lisa.util import LisaException
+
+from .git import Git
+
+
+class FIOResult:
+    qdepth: int = 0
+    mode: str = ""
+    iops: Decimal = Decimal(0)
+    latency: Decimal = Decimal(0)
+    iodepth: int = 0
+
+
+FIOMODES = Enum(
+    "FIOMODES",
+    [
+        "randread",
+        # "randwrite",
+        # "read",
```

Comment:
What's the reason to comment them out? If they are not necessary, just remove them.

Reply:
uncommented, it is for testing.